package.json

```diff
@@ -92,7 +92,7 @@
     "date-fns-tz": "^2.0.0",
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
-    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#35bff866a8d345b460ea6256f0a0f0a8a7f81086",
+    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#1609f1848cc0c2528064519c3ea48b4953a708ee",
```

Comment:
why this change?

Reply:
@BeeMargarida I needed the latest version of the lib with fixed Parameters type to finish the migration.
See [this comment](https://github.com/Expensify/App/issues/24865#issuecomment-1727343057) and [this PR](https://github.com/Expensify/expensify-common/pull/579)

Reply:
Got it, makes complete sense, ignore this comment